lib/clientsidescripts.js

```diff
@@ -599,15 +603,36 @@ functions.testForAngular = function(attempts, ng12Hybrid, asyncCallback) {
       asyncCallback(args);
     }, 0);
   };
+  var definitelyNg1 = !!ng12Hybrid;
+  var definitelyNg2 = false;
   var check = function(n) {
     try {
-      if (!ng12Hybrid && window.getAllAngularTestabilities) {
-        callback({ver: 2});
-      } else if (window.angular && window.angular.resumeBootstrap) {
-        callback({ver: 1});
-      } else if (n < 1) {
-        if (window.angular) {
+      // Figure out which version of angular we're waiting on
+      if(!definitelyNg1 && !definitelyNg2) {
+        if (window.angular && !(window.angular.version && window.angular.version.major > 1)) {
+          definitelyNg1 = true;
+        } else if (window.getAllAngularTestabilities) {
+          definitelyNg2 = true;
+        }
+      }
+      // See if our version of angular is ready
+      if (definitelyNg1) {
+        if (window.angular && window.angular.resumeBootstrap) {
+          return callback({ver: 1});
+        }
+      } else if (definitelyNg2) {
+        if (true /** ng2 has no resumeBootstrap() **/) {
+          return callback({ver: 2});
```

Comment:
Leaving this as `ver: 2` is fine for now but we should make this more future-proof later.